src/js/const/ticker-interval.const.js

```diff
@@ -0,0 +1 @@
+export const RELOAD_DATA_TICKER_INTERVAL_MS = 10000
```

Comment:
Move it to config.js pls, it may become handy in instances configurations

Reply:
ok